src/migration/1704948666033-GhostUserProtect.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class GhostUserProtect1704948666033 implements MigrationInterface {
+    name = 'GhostUserProtect1704948666033'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query(`CREATE RULE prototect_ghostuser_deletion AS ON DELETE TO "user" WHERE old.id IN ('404') DO INSTEAD nothing;`);
+      await queryRunner.query(`CREATE RULE prototect_ghostuser_update AS ON UPDATE TO "user" WHERE old.id IN ('404') DO INSTEAD nothing;`);
```

Comment:
The `UPDATE` rule is optional, as it might cause some headaches in future migrations if we want to modify something on all users, and forget about the rule.

Reply:
Yeah update is fine to remove it think, we sometimes change default image etc.

Reply:
```suggestion
```

Reply:
Agree, let's remove it. Especially if we want to make it a little bit more funny like you suggested

Reply:
Removed it.